io.openems.edge.ess.sma/src/io/openems/edge/ess/sma/stpxx3se/battery/Config.java

```diff
@@ -0,0 +1,37 @@
+package io.openems.edge.ess.sma.stpxx3se.battery;
+
+import org.osgi.service.metatype.annotations.AttributeDefinition;
+import org.osgi.service.metatype.annotations.ObjectClassDefinition;
+
+@ObjectClassDefinition(//
+		name = "ESS SMA Sunny Tripower SE Battery", //
+		description = "Collects Some more Sunspec unrelated data.")
+@interface Config {
+
+	@AttributeDefinition(name = "Component-ID", description = "Unique ID of this Component")
+	String id() default "battery0";
+
+	@AttributeDefinition(name = "Alias", description = "Human-readable name of this Component; defaults to Component-ID")
+	String alias() default "";
+
+	@AttributeDefinition(name = "Is enabled?", description = "Is this Component enabled?")
+	boolean enabled() default true;
+
+	@AttributeDefinition(name = "Modbus-ID", description = "ID of Modbus bridge.")
+	String modbus_id() default "modbus0";
+
+	@AttributeDefinition(name = "Modbus-Unit-ID", description = "Modbus Unit-ID.")
+	int modbusUnitId() default 3;
+	
+	@AttributeDefinition(name = "Charge Max Voltage", description = "Maximum voltage for charging")
```

Comment:
please add unit to description :  Maximum voltage for charging in V. 
Same for dischargeMinVoltage

Reply:
Okay, hab es geändert.